dashboard/src/components/ClassList/ClassList.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+
+import { IClusterServiceClass } from "../../shared/ClusterServiceClass";
+import { IRBACRole } from "../../shared/types";
+import Card, { CardContent, CardFooter, CardGrid, CardIcon } from "../Card";
+import { ErrorSelector, MessageAlert } from "../ErrorAlert";
+import LoadingWrapper from "../LoadingWrapper";
+import PageHeader from "../PageHeader";
+
+export interface IClassListProps {
+  error: Error | undefined;
+  classes: {
+    isFetching: boolean;
+    list: IClusterServiceClass[];
+  };
+  getClasses: () => void;
+}
+
+const RequiredRBACRoles: IRBACRole[] = [
+  {
+    apiGroup: "servicecatalog.k8s.io",
+    clusterWide: true,
+    resource: "clusterserviceclasses",
+    verbs: ["list"],
+  },
+];
+
+export default class ClassList extends React.Component<IClassListProps> {
+  public componentDidMount() {
+    this.props.getClasses();
+  }
+
+  public render() {
+    const { error, classes } = this.props;
+    return (
+      <div>
+        <PageHeader>
+          <h1>Classes</h1>
+        </PageHeader>
+        <main>
+          <LoadingWrapper loaded={!classes.isFetching}>
+            {error ? (
+              <ErrorSelector
+                error={error}
+                action="list"
+                defaultRequiredRBACRoles={{ list: RequiredRBACRoles }}
+                resource="Service Classes"
+              />
+            ) : (
+              <p>Types of services available from all brokers</p>
+            )}
+            {classes.list.length === 0 ? (
+              <MessageAlert level={"warning"}>
+                <h5>Unable to find any class.</h5>
```

Comment:
I think we should change this to "Service Classes not found" and then add some explanation on what happened, e.g. maybe pointing to this documentation: https://kubernetes.io/docs/concepts/extend-kubernetes/service-catalog/#usage